azure-pipelines.yml

```diff
@@ -30,7 +30,7 @@ jobs:
 
   - script: |
       pip install .
-      nosetests ./nilearn -v
+      pytest ./nilearn -v
```

Comment:
this was the case before so don't change it in this PR but unlike on travis, on azure the tests are run from the source dir

Reply:
Yeah, there are deeper issues with Azure (failing tests) which I will work on in a separate PR. I will do that once the current batch of my PRs are merged.

Reply:
sounds good